### PR TITLE
Update cloud-getting-started.md

### PR DESCRIPTION
**Pull request by tomtruitt**
for the noobs... probably still a better way to do this i'm sure

### CODE REVIEW

File: tutorials/advanced/cloud-getting-started.md [hunk 1]
```diff
@@ -28,30 +28,30 @@ Should you have no credentials or prefer not to utilize **any** cloud provider,
 
 The final dependency is that this tutorial will assume that you have added the proper credentials for whatever docker-machine driver you are using as environment variables. If you have not added them as environment variables then please add the necessary flags for the driver you will be using.
 
-For Digital Ocean the environment variables which should be set are:
+For Digital Ocean to set the environment variables:
```
Comment (compleatang) [inline tutorials/advanced/cloud-getting-started.md]: `to set` -> `set` (your active voice def better there)

File: tutorials/advanced/cloud-getting-started.md [hunk 2]
```diff
@@ -28,30 +28,30 @@ Should you have no credentials or prefer not to utilize **any** cloud provider,
...
 
-```irc
-DIGITALOCEAN_ACCESS_TOKEN=your_key
-DIGITALOCEAN_REGION=your_region
+```bash
+$ export DIGITALOCEAN_ACCESS_TOKEN=your_key
```
Comment (compleatang) [inline tutorials/advanced/cloud-getting-started.md]: we don't use the `$` as it makes copying and pasting not work well.

**Comment by compleatang** [comment]
Tom, those need some updates before I merge. Let me know if you'd like to resubmit or I can make them and push. Thanks for the edits!

**Comment by tomtruitt** [comment]
lol well i don't think i'll do better with this PR without more study and i have to keep moving forward....

**Comment by tomtruitt** [comment]
sorry might have got carried away unnecessary additions

**Comment by compleatang** [comment]
Tom, tons of commits here. Can you squash these and resubmit so we know what is happening.

**Comment by compleatang** [comment]
Tom, am going to close this PR for now.

**Pull request closed without merging**